src/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$tablet-min-width: 576px;
+$desktop-min-width: 1024px;
+
+$effectDuration: 0.25s;
+$menuScrollingDuration: 0.45s;
```

Use one style of writing variables.
```suggestion
$tablet-min-width: 576px;
$desktop-min-width: 1024px;

$effect-duration: 0.25s;
$menu-scrolling-duration: 0.45s;
```